### PR TITLE
cdc: add timeout to kafka docs.

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -218,6 +218,10 @@ URI 中可配置的的参数如下：
 | `sasl-user` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的用户名（authcid）（可选） |
 | `sasl-password` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的密码（可选） |
 | `sasl-mechanism` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的名称（可选） |
+| `dial-timeout` | 和下游 kafka 建立链接的超时时长，默认值为 `10s` |
+| `read-timeout` | 读取下游 kafka 返回的 response 的超时时长，默认值 `10s` |
+| `write-timeout`| 向下游 kafka 发送 request 的超时时长，默认值为 `10s` |
```
Comment (qiancai) [inline ticdc/manage-ticdc.md:223]:
```suggestion
| `dial-timeout` | 和下游 Kafka 建立连接的超时时长，默认值为 `10s` |
| `read-timeout` | 读取下游 Kafka 返回的 response 的超时时长，默认值 `10s` |
| `write-timeout`| 向下游 Kafka 发送 request 的超时时长，默认值为 `10s` |
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -218,6 +218,10 @@ URI 中可配置的的参数如下：
 | `sasl-user` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的用户名（authcid）（可选） |
 | `sasl-password` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的密码（可选） |
 | `sasl-mechanism` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的名称（可选） |
+| `dial-timeout` | 和下游 Kafka 建立连接的超时时长，默认值为 `10s` |
+| `read-timeout` | 读取下游 Kafka 返回的 response 的超时时长，默认值 `10s` |
+| `write-timeout`| 向下游 Kafka 发送 request 的超时时长，默认值为 `10s` |
+
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:224]:
```suggestion
| `read-timeout` | 读取下游 Kafka 返回的 response 的超时时长，默认值为 `10s` |
| `write-timeout`| 向下游 Kafka 发送 request 的超时时长，默认值为 `10s` |
```

**Review by shichun-0415 — APPROVED** [review]

File: ticdc/manage-ticdc.md (hunk 2 above)
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:224]:
```suggestion
```

**Review by qiancai — APPROVED** [review]
LGTM

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -218,7 +218,9 @@ URI 中可配置的的参数如下：
 | `sasl-user` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的用户名（authcid）（可选） |
 | `sasl-password` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的密码（可选） |
 | `sasl-mechanism` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的名称（可选） |
-
+| `dial-timeout` | 和下游 Kafka 建立连接的超时时长，默认值为 `10s` |
+| `read-timeout` | 读取下游 Kafka 返回的 response 的超时时长，默认值为 `10s` |
+| `write-timeout`| 向下游 Kafka 发送 request 的超时时长，默认值为 `10s` |
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:223]:
```suggestion
| `write-timeout`| 向下游 Kafka 发送 request 的超时时长，默认值为 `10s` |

```

**Comment by shichun-0415** [comment]
@3AceShowHand Could you please involve a technical reviewer? Thanks.

**Review by overvenus — APPROVED** [review]
LGTM

**Pull request merged**